### ISSUE TITLE
Unable to compile 0.9.4 64bit on ubuntu 16.10

### ISSUE THREAD

**Issue by jochil** [openframeworks/openFrameworks#5318]
I am trying to install OF 0.9.4 on Ubuntu 16.10 but calling the compileOF.sh script fails with the following message
```
/home/jochen/tools/of/libs/openFrameworks/sound/ofRtAudioSoundStream.cpp:8:21: fatal error: RtAudio.h: No such file or directory
 #include "RtAudio.h"
                     ^
compilation terminated.
makefileCommon/compile.core.mk:239: recipe for target '/home/jochen/tools/of/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/sound/ofRtAudioSoundStream.o' failed
make[1]: *** [/home/jochen/tools/of/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/sound/ofRtAudioSoundStream.o] Error 1
make[1]: *** Waiting for unfinished jobs....
makefileCommon/compile.core.mk:213: recipe for target 'Debug' failed
make: *** [Debug] Error 2
there has been a problem compiling Debug OF library
please report this problem in the forums
```
During the dependency installation it offers to recompile peco, which finished without problems

**Comment by arturoc** [comment]
at the beginning of the script output there should be something like:

```
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libcurl liburiparser glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
```

can you post what you are getting there?

**Comment by jochil** [comment]
of course
```
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123
```

**Comment by arturoc** [comment]
i thought this was already fixed in 0.9.4 but it's not, i'm preparing a 0.9.5 release with this and some other fixes for ubuntu 16.10 and qtcreator on osx
